oauth2-authorization-server/src/test/java/org/springframework/security/config/annotation/web/configurers/oauth2/server/authorization/OAuth2ClientCredentialsGrantTests.java

```diff
@@ -140,6 +143,38 @@ public void requestWhenTokenRequestPostsClientCredentialsThenTokenResponse() thr
 		verify(authorizationService).save(any());
 	}
 
+	@Test
+	public void requestWhenCustomProviderSettingsThenOk() throws Exception {
```

Comment:
Let's remove these tests since it will be covered in `OAuth2AuthorizationCodeGrantTests` (see previous comment)